### PR TITLE
[1.13.x] [KIECLOUD-654] - bundle replaces points to wrong previous version

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

File: modules/org.kie.kogito.rhpam.builder/module.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: org.kie.kogito.rhpam.builder
-version: "7.11.0"
```
Comment (rgdoliveira) [inline modules/org.kie.kogito.rhpam.builder/module.yaml:2]: I believe here it should be 7.13.1? The rest of the PR looks good.

File: modules/org.kie.kogito.rhpam.builder/module.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+version: "7.13.0"
```
Comment (rgdoliveira) [inline modules/org.kie.kogito.rhpam.builder/module.yaml:2]: I believe here it should be 7.13.1? The rest of the PR looks good.

**Review by radtriste — APPROVED** [review]
ok once comment from @rgdoliveira is addressed

**Review by rgdoliveira — APPROVED** [review]
LGTM

File: modules/org.kie.kogito.rhpam.builder/module.yaml (hunk 2 above)
Comment (radtriste) [reply modules/org.kie.kogito.rhpam.builder/module.yaml:2]: does not seem to be replaced ...

**Comment by radtriste** [comment]
@spolti `note: module requires Go 1.17`, this looks weird ...

**Comment by spolti** [comment]
> @spolti `note: module requires Go 1.17`, this looks weird ...

yes, I am not sure why.

**Comment by spolti** [comment]
Golang version will updated as part of https://github.com/kiegroup/kogito-operator/pull/1280

**Comment by rgdoliveira** [comment]
@radtriste seems this PR was merged without applying version change in modules/org.kie.kogito.rhpam.builder/module.yaml to 7.13.1. I've opeend https://github.com/kiegroup/kogito-operator/pull/1285 to fix it.

**Comment by radtriste** [comment]
> @radtriste seems this PR was merged without applying version change in modules/org.kie.kogito.rhpam.builder/module.yaml to 7.13.1. I've opened #1285 to fix it.

@rgdoliveira this is an old PR that we did not merge until now ... thanks for the fix

**Pull request merged**